syntax = "proto3";

message RequestOperation{
    string data = 1;
}

message ResponseOfOperation{
    string data = 1;
}

service GetOperation{
    rpc addToClientDB (RequestOperation) returns (ResponseOfOperation) {}
}